.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
```

Comment:
Suggest changing this to sub headings, i.e.

```suggestion
#### Description
```

Same for the others, reason being it reduces the size of the comment at the head of the PR, allowing greater visibility over the content of the PR which is 'more' important

Reply:
Comment for this PR update to show the smaller headings